src/sim/UI/view.py

```diff
@@ -4,157 +4,74 @@
-        self.openGrid.setStyleSheet("background-color: white;")
-        self.radbtn_1 = QRadioButton('Radio1', self.__window)
-        self.radbtn_1.setStyleSheet("color: white;")
-        self.radbtn_2 = QRadioButton('Radio2', self.__window)
-        self.radbtn_2.setStyleSheet("color: white;")
-        self.radbtn_3 = QRadioButton('Radio3', self.__window)
-        self.radbtn_3.setStyleSheet("color: white;")
-        self.radbtn_4 = QRadioButton('Radio4', self.__window)
-        self.radbtn_4.setStyleSheet("color: white;")
-        self.chkbtn_1 = QCheckBox('Check1', self.__window)
-        self.chkbtn_1.setStyleSheet("color: white;")
-        self.chkbtn_2 = QCheckBox('Check2', self.__window)
-        self.chkbtn_2.setStyleSheet("color: white;")
-        self.chkbtn_3 = QCheckBox('Check3', self.__window)
-        self.chkbtn_3.setStyleSheet("color: white;")
+        topLayout = QHBoxLayout()
+        bottomLayout = QVBoxLayout()
+        mainLayout = QVBoxLayout()
 
-        # Connected radio buttons to their corresponding action functions
-        self.radbtn_1.clicked.connect(self.radioButton1)
-        self.radbtn_2.clicked.connect(self.radioButton2)
-        self.radbtn_3.clicked.connect(self.radioButton3)
-        self.radbtn_4.clicked.connect(self.radioButton4)
+        # Create IP Label and Textbox
+        ipLabel = QLabel('IP Address')
+        ipTextbox = QLineEdit()
 
-        # Connected check boxes to their corresponding action functions
-        self.chkbtn_1.clicked.connect(self.checkBox1)
-        self.chkbtn_2.clicked.connect(self.checkBox2)
-        self.chkbtn_3.clicked.connect(self.checkBox3)
+        # Make Push Button
+        newRobotButton = QPushButton("Add Robot")
 
-        # add radio widgets to radio layout
-        radio_layout.addWidget(self.radbtn_1)
-        radio_layout.addWidget(self.radbtn_2)
-        radio_layout.addWidget(self.radbtn_3)
-        radio_layout.addWidget(self.radbtn_4)
-        radio_layout.addStretch(5)
-        radio_layout.setSpacing(10)
-        radio_widget = QWidget()
-        radio_widget.setLayout(radio_layout)
-        radio_widget.setStyleSheet("background-color: black;")  # Set background color
-        radio_widget.setFixedHeight(50)
+        # Bind Push Button to Event Handler
+        newRobotButton.clicked.connect(self.__addRobotButtonClicked)
 
-        
-        # add checkbox widgets to checkbox layout
-        checkbox_layout.addWidget(self.chkbtn_1)
-        checkbox_layout.addWidget(self.chkbtn_2)
-        checkbox_layout.addWidget(self.chkbtn_3)
-        checkbox_layout.addStretch(5)
-        checkbox_layout.setSpacing(10)
-        checkbox_widget = QWidget()
-        checkbox_widget.setLayout(checkbox_layout)
-        checkbox_widget.setStyleSheet("background-color: black;")  # Set background color
-        checkbox_widget.setFixedHeight(50)
+        # Add IP Label, IP Textbox, New Robot Button to Top Layout
+        topLayout.addWidget(ipLabel)
```

Comment:
What is the point of implementing the IP here?

Reply:
I guess we could just increment the IPs. so robot 0 has ip 10.0.0.10 and robot 1 has IP 10.0.0.11 etc. doesnt matter to me, what do you think is betteer

Reply:
I am not sure I understand the purpose of having the IP for the simulation. Is it for something that can't be virtualized?

Reply:
the ip is required.

Reply:
What is it required for outside of the hardware and TCP?

Reply:
i cant tell if this is a joke or not...

Reply:
ok, we'll discuss tomorrow then